Tests/UAPusherTests/AudienceTests.swift

```diff
@@ -0,0 +1,33 @@
+import XCTest
+import JSON
+@testable import UAPusher
+
+class AudienceTests: XCTestCase {
+    
```

Comment:
We need to add the `allTests` property and then `LinuxMain.swift` needs to be updated with the new test files. This is required for it to run on Linux.